tests/pipelines/pnc.nf.test

```diff
@@ -0,0 +1,1335 @@
+            assert cluster_summary.text.contains("1.1.1.1\t1,nan\t2501STWGS-01,nan\tclosed,nan\t2025-01-10\t2025-01-17\tSalmonella enterica subsp. enterica\tTyphimurium\t10\t2.0\t89.0\t78.93333333333334\t135.0\t1\t0_outgroup_9\t2025-01-01\t2025-01-10\tAnimal,Environmental,Food,Human,Unknown\t2\t1\t2\t4\t1\tAB,BC,MB,NB,NL,NS,ON,PE,QC,SK\t1\t1\t1\t1\t1\t1\t0\t0\t1\t1\t1\t1\t0")
+            assert cluster_summary.text.contains("1.1.2.2\t2,nan\t2502SEWGS-01,nan\tclosed,nan\t2025-02-07\t2025-04-04\tSalmonella enterica subsp. enterica\tEnteritidis\t10\t10.0\t79.0\t74.77777777777777\t140.0\t1\t1_outgroup_19\t2025-02-01\t2025-03-11\tAnimal,Environmental,Food,Human\t2\t1\t1\t6\t0\tAB,BC,MB,NT,NU,SK,YK\t1\t2\t2\t0\t0\t0\t1\t1\t0\t0\t0\t2\t1")
+            assert cluster_summary.text.contains("10.10.10.10\t3,nan\t2503SHWGS-01,nan\tclosed,nan\t2025-03-07\t2025-04-11\tSalmonella enterica\tHeidelberg\t10\t6.0\t86.0\t75.8\t134.0\t0\t\t2025-03-17\t2025-04-01\tAnimal,Environmental,Food,Human\t2\t2\t3\t3\t0\tNB,NL,NS,ON,PE,QC\t0\t0\t0\t2\t1\t1\t0\t0\t3\t1\t2\t0\t0")
+            assert cluster_summary.text.contains("2.4.4.4\t4,nan\t2504SAL-01,nan\tclosed,nan\t2025-04-11\t2025-05-23\tSalmonella arizonae,Salmonella enterica\tIIIa 51:z36:-\t10\t2.0\t78.0\t71.0\t124.0\t0\t\t2025-04-05\t2025-05-10\tAnimal,Environmental,Food,Human\t2\t2\t1\t5\t0\tNB,NL,NS,NT,PE,QC,YK\t0\t0\t0\t2\t1\t1\t1\t0\t0\t1\t2\t0\t2")
+            assert cluster_summary.text.contains("5.5.5.21000\t5,nan\t2505SAL-01,nan\topen,nan\t2025-05-09\t2025-06-06\tSalmonella enterica,Salmonella arizonae\tI 1,2,12:a:[1,5]|I 1,2,12:a:-\t10\t2.0\t93.0\t75.77777777777777\t138.0\t1\t4_outgroup_49\t2025-05-12\t2025-05-31\tAnimal,Environmental,Food,Human\t2\t3\t3\t2\t0\tAB,BC,NL,NS,NT,NU,PE,YK\t2\t1\t0\t0\t1\t1\t1\t2\t0\t1\t0\t0\t1")
+            assert cluster_summary.text.contains("1.3.3.3\t6,nan\t2506SAL-01,nan\tclosed,nan\t2025-06-06\t2025-08-08\tSalmonella enterica\tI 1,4,12,27:z41:1,(2),5\t10\t3.0\t83.0\t73.97777777777777\t132.0\t1\t5_outgroup_59\t2025-06-02\t2025-07-06\tAnimal,Environmental,Food,Human\t2\t1\t2\t5\t0\tAB,BC,NL,NS,NT,NU,PE,SK,YK\t1\t1\t0\t0\t1\t1\t1\t2\t0\t1\t0\t1\t1")
```

Comment:
A very small issue I observed. Some of the serotypes like `I 1,4,12,27:z41:1,(2),5` contain commas. But in the cluster summary file, commas are also used to separate multiple values. This could lead to uncertainty about how to split apart these values again.

But, the cluster summary file is meant mainly to be read by people, who will be aware of different serotypes and so could sort out which commas are part of a serotype and which are meant to split multiple serotypes. So I don't think this is something we need to address now. Just wanted a note of this for later. This is also something that needs to be addressed in the arborator Python package rather than the pipeline.